cpp/src/interop/to_arrow.cpp

```diff
@@ -101,17 +101,15 @@ struct dispatch_to_arrow {
     rmm::cuda_stream_view stream)
   {
     std::vector<std::shared_ptr<arrow::Array>> child_arrays;
-    std::vector<size_type> child_indices(input_view.num_children());
-    std::iota(child_indices.begin(), child_indices.end(), 0);
-    std::transform(child_indices.begin(),
-                   child_indices.end(),
-                   metadata.begin(),
-                   std::back_inserter(child_arrays),
-                   [&input_view, &ar_mr, &stream](auto const& i, auto const& meta) {
-                     auto c = input_view.child(i);
-                     return type_dispatcher(
-                       c.type(), dispatch_to_arrow{}, c, c.type().id(), meta, ar_mr, stream);
-                   });
+    std::transform(
```

Comment:
swoon

Reply:
This is thanks to you Jake :fire:  We should briefly mention `child_begin/end` because I wasn't aware of them :slightly_smiling_face: